### ISSUE TITLE
Error Ban.Profile#getCreationDate() 

### ISSUE THREAD

**Issue by rexbut** [SpongePowered/Sponge#1129]
Ban.Profile#getCreationDate() return the value of Ban.Profile#getExpirationDate().

There is an error in the UserListBansEntry :
![Error](http://image.noelshack.com/fichiers/2017/02/1484259647-ban.png)

**Comment by simon816** [comment]
Looks like a mistake in vanilla.
I quickly tested by getting the vanilla server to convert an old `banned-players.txt` file to the new format because it calls this constructor.
```
simon8162|2016-01-01 12:00:00 GMT|admin|2018-01-01 12:00:00 GMT|reason
```

```json
[
  {
    "uuid": "48708698-b1b1-43f2-908d-5383bbaa27bb",
    "name": "simon8162",
    "created": "2018-01-01 12:00:00 +0000",
    "source": "admin",
    "expires": "2018-01-01 12:00:00 +0000",
    "reason": "reason"
  }
]
```
"created" should be `2016-01-01 12:00:00 +0000` but is instead `2018-01-01 12:00:00 +0000`

**Comment by lesbleu** [comment]
I opened an issue on Mojang.
https://bugs.mojang.com/browse/MC-112684

**Comment by Mumfrey** [comment]
This issue can now be resolved using Mixin `0.6.12` using the new [ModifyArgs](http://jenkins.liteloader.com/view/Other/job/Mixin/javadoc/index.html?org/spongepowered/asm/mixin/injection/ModifyArgs.html) injector.
